paddle/fluid/operators/math/depthwise_conv.cu

```diff
@@ -142,13 +141,14 @@ __device__ __inline__ void KernelDepthwiseConvNHWC(
     for (int w_in = w_in_start; w_in < w_in_end; w_in += dilate_width) {
       if (h_in >= h_start && h_in < h_end && w_in >= w_start && w_in < w_end) {
         int offset = ((batch * input_height + h_in) * input_width + w_in) *
-                         output_channels +
+                         input_channels +
                      c_in;
         T in_data = input_data[offset];
+        const T* weight = filter_data + weight_offset * output_channels + c_out;
         if (fuse_relu_before_conv) {
-          value += weight[weight_offset] * max(0.0f, in_data);
+          value += weight[0] * max(0.0f, in_data);
         } else {
-          value += weight[weight_offset] * in_data;
+          value += weight[0] * in_data;
```

Comment:
Could you describe why this change was made?

Reply:
Filter_data was transposed from `CHW` to `HWC` here, so weight in `(h_f, w_f, c_out)` should be `const T* weight = filter_data + weight_offset * output_channels + c_out`,  which `weight_offset` equals to `h_f * filter_width + w_f`.